test/src/index/testNightwatchIndex.js

```diff
@@ -198,11 +182,6 @@ describe('test NightwatchIndex', function () {
 
     client.startSession().catch(err => {
       assert.ok(err instanceof Error);
-      assert.strictEqual(typeof err.data, 'string');
-      assert.deepStrictEqual(JSON.parse(err.data), {
-        message: 'Could not find device : iPhone 6',
-        error: []
-      });
```

Comment:
Any reason to remove these assertions?

Reply:
There is no data for error object in v2.